terraform/fullnode/vultr/.terraform.lock.hcl

```diff
@@ -4,6 +4,7 @@
 provider "registry.terraform.io/hashicorp/helm" {
   version = "2.5.1"
   hashes = [
+    "h1:9yMFsXyHAo+mUuMKczNSw44HcZaf1JkMqgOUgJF1dXs=",
```

Comment:
we can probably leave out these terraform lockfiles, since they're meant to be invoked as modules